en/frames_multicopter/qav_r_5_kiss_esc_racer.md

```diff
@@ -217,30 +222,62 @@ I mounted it like you can see in the next picture. The two screws and nuts to lo
 
 ![Camera](../../assets/airframes/multicopter/qav_r_5_kiss_esc_racer/fpv-cam.jpg)
 
-## Software Configuration
+## PX4 Configuration
+
+*QGroundControl* is used to install the PX4 autopilot and configure/tune it for the frame.
+[Download and install](http://qgroundcontrol.com/downloads/) *QGroundControl* for your platform.
+
+:::tip
+Full instructions for installing and configuring PX4 can be found in [Basic Configuration](../config/README.md).
+:::
 
 :::warning
 Always make sure to have either battery or propellers physically removed from your vehicle during any initial configuration.
 Better safe than sorry!
 :::
 
-For general configuration instruction please see [Basic Configuration](../config/README.md).
 
-For this build I pulled the latest PX4 master because it supports the "FMU as task" improvements (included and enabled by default since PX4 v1.7) and flashed it to the Pixracer. I used [QGC](http://qgroundcontrol.com/) daily build to configure the following:
-- Choose the [Generic 250 Racer configuration](../airframes/airframe_reference.md#copter_quadrotor_x_generic_250_racer) Airframe
-- Calibrate the sensors
-- Set the battery to 4S (4 cell LiPo) with charged cell voltage 4.15V and empty cell voltage 3.5V
-- Calibrate the voltage divider through typing in the current accurate voltmeter measurement
-- Calibrate the RC cannels with the Taranis already configured for two additional switch inputs.
-  One switch in the top right corner of the Taranis front plate for the mode switch and the other switch in the top left corner of the front plate as arm switch.
-- You can also set up an arming switch.
+First update the firmware and airframe:
+
+* [Firmware](../config/firmware.md)
+* [Airframe](../config/airframe.md)
+
+  You will need to select the *Generic 250 Racer* airframe (**Quadrotor x > Generic 250 Racer**).
+
+  ![QGC airframe selection of generic 250 racer](../../assets/airframes/multicopter/qav_r_5_kiss_esc_racer/qgc_airframe_generic_250_racer.png)
+
+Then set the geometry and outputs
+
+- [Actuators](../config/actuators.md)
+  - You should not need to update the vehicle geometry (as this is a preconfigured airframe).
+  - Assign actuator functions to outputs to match your wiring
+  - Test the configuration using the sliders. 
```

Comment:
Hi @bkueng I've been updating all the build logs to have a more or less common PX4 configuration section. 

Originally I just put you should not have to configure Actuators for these things that have a canned airframe. However trying this out it looks like the geometry is set, but the outputs are never mapped to specific motors/servos by default - you always have to do that.

So my understanding is that this is what I should say in all of the "specific" airframes. For a "generic airframe" I would have to also mention setting the geometry.

That about right?

Reply:
Partially, we're not using a specific airframe here. It's still generic in the sense that it's for a class of vehicles.

Reply:
@bkueng Ah, curses - this is "generic". But for [X500 v2](http://docs.px4.io/main/en/frames_multicopter/holybro_x500V2_pixhawk5x.html#px4-configuration) you would not need to update the geometry because it should be set.

Reply:
Yes that one should include both, geometry & outputs.